### DISCUSSION TITLE
How to Pick up Playlists from ConfigMap

### DISCUSSION THREAD

**Discussion by mvkrishna86** [grafana/grafana#72917] [Dashboards, Panels & Visualizations]
We have setup grafana OSS in kubernetes cluster. I am managing all the dashboards from configmap and configured the setup using flux.
Is there any way we can configure the playlists in json or any other files? so that I can create configmaps and manage using gitops?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82545. Feel free to continue the discussion around this there. Thank you!
